src/Form/TelemetryOptInForm.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\lightning\Form\TelemetryOptInForm.
+ */
+namespace Drupal\lightning\Form;
+
+use Drupal\Core\Extension\ModuleInstallerInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class TelemetryOptInForm.
+ *
+ * This form is displayed during the Lightning install process. It provides
```

Comment:
I don't see this form being surfaced anywhere _after_ initial installation, so we're not likely to get any pre-existing users to opt in. Should we add some kind of `hook_requirements()` implementation that prompts the user to install the module or hide the message?

Reply:
It's a good idea. Maybe we should, but I wouldn't consider it MVP.

Reply:
One option is to add a hook_update_N() which simply sets a message asking users to enable the new module. That'd be very easy, and unobtrusive.